apps/cyberstorm-remix/cyberstorm/security/publicEnvVariables.ts

```diff
@@ -47,11 +47,10 @@ export function getPublicEnvVariables(
   return returnedVars;
 }
 
-export function getSessionTools() {
-  const publicEnvVariables = getPublicEnvVariables([
-    "VITE_API_URL",
-    "VITE_COOKIE_DOMAIN",
-  ]);
+export function getSessionTools(preFetchedEnv?: publicEnvVariablesType) {
```

Comment:
Not an issue but is there some other reason for the new argument than trying to save a couple ms when entry.client.tsx executes? Might be more noise in the code than practical value in performance 🤷🏻